great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -186,7 +193,20 @@ def _get_dialect_type_module(dialect):  # noqa: C901
             and teradatatypes is not None
         ):
             return teradatatypes
-    except (TypeError, AttributeError):
+    except (TypeError, AttributeError, ModuleNotFoundError):
+        pass
+
+    # Clickhouse types module
+    try:
```

Comment:
Confirming understanding here. Were we not correctly checking whether the `Clickhouse` module was imported successfully before?